examples/nucleo_f401re/adc_dma/main.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2023, Zühlke (Austria) gmbh
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// This Example uses adc to sample 2 channels in scan mode 50 times every 0.5s and store the value
+// in a buffer via dma. The adc is also set to be triggered by Timer1 compare Event, that is set in
```

Comment:
via dma. <linebreak>
The adc is set to be triggered by the Timer1 compare event and than samples both channels using scan mode. The compare event is triggered for a defined number of times by configuring the Timer1 to use one pulse mode with an repetition count. <linebreak>
For more information...